schema/slots/undefined-slot.json

```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Undefined Slot",
+    "description": "A slot for an unknwown property type",
+    "properties": {
+        "t": {
+            "title": "Slot Type",
+            "const": 99
```

Comment:
Do we need to define a constant for these or assume any `t` not in the specs is an unknown type?

Reply:
Typing info removed from 1.0 slot spec